doc/classes/Rect2.xml

```diff
@@ -4,10 +4,10 @@
 		A 2D axis-aligned bounding box using floating-point coordinates.
 	</brief_description>
 	<description>
-		[Rect2] consists of a position, a size, and several utility functions. It is typically used for fast overlap tests.
-		It uses floating-point coordinates. If you need integer coordinates, use [Rect2i] instead.
-		The 3D counterpart to [Rect2] is [AABB].
-		Negative values for [member size] are not supported and will not work for most methods. Use [method abs] to get a Rect2 with a positive size.
+		The [Rect2] built-in [Variant] type represents a rectangle in 2D space. It is defined by its [member position] and [member size], which are [Vector2]. Because it does not rotate, it is frequently used for fast overlap tests (see [method intersects]).
```

Comment:
I don't see a problem with "2D space". If you follow this suggestion, don't forget to also add an "a" article before "2D coordinate system"

Reply:
I suppose that "2D space" could be mistook as something along the lines of "2D world space" which is how global coordinates may be defined. I understand it, but I'd still favour "2D space" because it does not make it sound as complicated.

Reply:
Actually, I though "coordinate system" is preferred over "space" across the docs, that's pretty much it. :upside_down_face: Probably I had such feeling after https://github.com/godotengine/godot-docs/pull/6277.  But after searching for "2D space" there are a lot other places referring as such. So, if anything, these could be changed/unified separately in another PR if desired (either for consistency or for differentiating 2D space as a whole vs "specific view at such space" / specific coordinate system).

So yeah, "2D space" is fine too.
(I wouldn't skip the "axis-aligned" part though, even if it's already mentioned in the brief description.)

>If you follow this suggestion, don't forget to also add an "a" article before "2D coordinate system"

Hmm, isn't it still needed before "2D space" though? :thinking:

Reply:
The "a" article is a subtle nudge, I feel. It helps convey that it's a generic space unless the developer gives it context.